docs/community-clients/index.asciidoc

```diff
@@ -1,223 +1,230 @@
-[[clients]]
-== Clients
+= Community Contributed Clients
 
+:client: https://www.elastic.co/guide/en/elasticsearch/client
 
-[[community-perl]]
-=== Perl
+Besides the link:/guide[officially supported Elasticsearch clients], there are
+a number of clients that have been contributed by the community for various languages:
 
-See the {client}/perl-api/current/index.html[official Elasticsearch Perl client].
-
-[[community-python]]
-=== Python
-
-See the {client}/python-api/current/index.html[official Elasticsearch Python client].
-
-* http://github.com/elasticsearch/elasticsearch-dsl-py[elasticsearch-dsl-py]
-  chainable query and filter construction built on top of official client.
-
-* http://github.com/rhec/pyelasticsearch[pyelasticsearch]:
-  Python client.
+* <<clojure>>, <<cold-fusion>>
```

Comment:
Yes - the list was just getting a bit long, but perhaps it'll be clearer to leave it as one-per-bullet